README.adoc

```diff
@@ -326,6 +326,31 @@ _build/features_report.html_. They can be opened via:
 $ bundle exec rake features:open_report
 ---------------------------------------
 
+=== Asciidoc
+
+The documentation of this plugin is written in http://asciidoctor.org[Asciidoc]. If you need some syntax help,
+refer to the http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc Syntax Quick Reference].
+
+To build the documentation you can execute
+
+----
+$ bundle exec rake html
+----
+
+which will build the HTML documentation into the folder _build/html_.
+
+The source code also contains a link:Guardfile[Guardfile] for the https://github.com/guard/guard[Guard] library.
+You can execute
+
+----
+$ bundle exec guard
+----
+
+and your HTML documentation will be automatically updated on each change to an Asciidoc source file.
+https://github.com/guard/guard-livereload[Live reload] is also enabled, so you just need to install the right
+http://livereload.com/extensions/#installing-sections[LiveReload Safari/Chrome/Firefox extension] and your
+browser will refresh the page each time you save a change to your Asciidoc files.
```

Comment:
@hferentschik , since this `live reload` is not working efficiently, do you mind updating this section of docs to be like an optional or addition so that user doesn't expect as I was expecting.